dir.targets

```diff
@@ -58,4 +58,17 @@
     </SupplementalTestData>
   </ItemGroup>
 
+  <!-- ToDo: Remove this target and the following ItemGroup once the mapping between netstandard1.7 and netcoreapp1.1 is supported -->
```

Comment:
> ToDo: Remove this ta [](start = 6, length = 21)

please make sure you have issue tracking this. and possibly add the issue number to this comment.